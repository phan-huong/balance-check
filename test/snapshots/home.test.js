const { app, request } = require('../commonJest')

it('renders courses page correctly', (done) => {
  request(app)
    .get('/')
    .then((response) => {
      expect(response.text).toMatchSnapshot()
      done()
    })
})
