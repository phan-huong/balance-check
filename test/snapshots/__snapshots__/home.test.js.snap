// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders courses page correctly 1`] = `
"<!DOCTYPE html>
<html lang=\\"en\\">
  <head>
    <meta charset=\\"utf-8\\">
    <link rel=\\"stylesheet\\" href=\\"css/balance-check.css\\">
    <script src=\\"js/balance-check.js\\"></script>
    <style>
      body{font-family: verdana; padding: 30px;}
      h1{font-size: 36px; text-transform: uppercase; letter-spacing: 2px; text-align: center;}
      h2{font-size: 24px; text-align: center;}
      div{margin: auto; font-size: 16px;}
      img{display: inline-block; border-radius: 10px; margin-left: auto; margin-right: auto; width:70%; max-width: 500px;}
      #header{font-family: verdana; font-size: 24px; text-align: center; padding: 10px; left: 0; right: 0; up: 0; width: 100%; color: grey;}
      #footer{position: fixed; text-align: center; left: 0; right: 0; bottom: 0; width: 100%; background-color: dodgerblue; color: white;}
    </style>
  </head>
  <body>
    <b><div id=\\"header\\">Balance Check</div></b>
    <!DOCTYPE html>
<html lang=\\"en\\" dir=\\"ltr\\">
  <head>
    <meta charset=\\"utf-8\\">
    <title>Home Page</title>
  </head>
  <body>
    <div class=\\"centerHorizontalVertical\\">
      <h1> HOME PAGE </h1>
      <form action=\\"/myAccount\\" method=\\"post\\">
        <input type=\\"text\\" name=\\"username\\">
        <button>Sign In</button>
      </form>
      <form action=\\"/users/new\\" method=\\"get\\">
        <button>Sign Up</button>
      </form>
    </div>
  </body>
</html>

    <div id=\\"footer\\"><p> FOOTER </p></div>
  </body>
</html>
"
`;
